tests/lib/collectors/gsheets.py

```diff
@@ -10,37 +10,10 @@
 from dftimewolf.lib.collectors import gsheets
 import mock
 
-INVALID_SHEET = {
-    'range':
-        'Sheet2!A1:Y1000',
-    'majorDimension':
-        'ROWS',
-    'values': [[
-        'message', 'timestamp', 'timestamp_desc ', 'extra_field_1 ',
-        'extra_field_2'
-    ], ['A message', '1331698658276340', 'Write time', 'foo ', 'bar'],
-               ['', '1331698658276340', 'create', 'dodo'],
-               ['sdsadasd', '', 'ddd', 'dodo', 'd']]
-}
-VALID_SHEET = {
-    'range':
-        'Sheet1!A1:Z1000',
-    'majorDimension':
-        'ROWS',
-    'values': [[
-        'message', 'timestamp', 'datetime', 'timestamp_desc ', 'extra_field_1 ',
-        'extra_field_2'
-    ],
-               [
-                   'A message', '1331698658276340', '2015-07-24T19:01:01+00:00',
-                   'Write time', 'foo ', 'bar'
-               ],
-               [
-                   '', '1331698658276340', '2016-07-24T19:01:01+00:00',
-                   'create', 'dodo'
-               ], ['sdsadasd', '', '', 'ddd', 'dodo', 'd']]
-}
 
+VALID_SHEET = {'range': 'Sheet1!A1:Z1000', 'majorDimension': 'ROWS', 'values': [['message', 'timestamp', 'datetime', 'timestamp_desc ', 'extra_field_1 ', 'extra_field_2'], ['A message', '1331698658276340', '2015-07-24T19:01:01+00:00', 'Write time', 'foo ', 'bar'], ['', '1331698658276340', '2016-07-24T19:01:01+00:00', 'create', 'dodo' ], ['sdsadasd', '', '', 'ddd', 'dodo', 'd']]} # pylint: disable=line-too-long
+# Missing "datetime" columnd
+INVALID_SHEET = {'range': 'Sheet2!A1:Y1000', 'majorDimension': 'ROWS', 'values': [['message', 'timestamp', 'timestamp_desc ', 'extra_field_1 ','extra_field_2'], ['A message', '1331698658276340', 'Write time', 'foo ', 'bar'],['', '1331698658276340', 'create', 'dodo'],['sdsadasd', '', 'ddd', 'dodo', 'd']]} # pylint: disable=line-too-long
```

Comment:
I guess my previous comment was not clear, I meant the actual sheet information (i.e. the rows in the spreadsheet) on the same line (to be sort of similar to what you would expect in a spreadsheet). The idea here is to make the contents of this variable easy to understand by humans.

Reply:
Got it now. Just did that.